log/log.go

```diff
@@ -41,6 +42,23 @@ func NewLogger(logLevel Level) *Logger {
 	return NewLogrusLogger(logrusLog)
 }
 
+// NewCronLogger creates an instance of *Logger.
+func NewCronLogger(logLevel Level) *Logger {
+	logrusLog := logrus.New()
+
+	// Configure logger
+	// Don't log to stderr by default (stackdriver treats stderr as error severity)
+	logrusLog.SetOutput(os.Stdout)
+	// for stackdriver, see: https://cloud.google.com/stackdriver/docs/solutions/gke/managing-logs
+	logrusLog.SetFormatter(&logrus.JSONFormatter{})
```

Comment:
do we need to customize anything by providing options?

i.e mapping log level to `severity`?
> Logs generated with glog will have the common fields parsed, for example, `severity`, `pid`, `source_file`, `source_line`